### PR TITLE
Moved SpanHelper to internal

### PR DESCRIPTION
**Pull request by utpilla**
## Changes
- Modified SpanHelper class and ResolveSpanStatusForHttpStatusCode method to internal
- Updated AspNet, AspNetCore and Http instrumentation projects to add SpanHelper.cs as a link

Addressing #1553

### CODE REVIEW

File: src/OpenTelemetry.Api/Internal/SpanHelper.cs [hunk 1]
```diff
@@ -19,15 +19,15 @@ namespace OpenTelemetry.Trace
...
+    internal static class SpanHelper
     {
         /// <summary>
         /// Helper method that populates span properties from http status code according
         /// to https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md#status.
         /// </summary>
         /// <param name="httpStatusCode">Http status code.</param>
         /// <returns>Resolved span <see cref="Status"/> for the Http status code.</returns>
-        public static Status ResolveSpanStatusForHttpStatusCode(int httpStatusCode)
+        internal static Status ResolveSpanStatusForHttpStatusCode(int httpStatusCode)
```
Comment (cijothomas) [inline src/OpenTelemetry.Api/Internal/SpanHelper.cs:30]: As class is anyway internal, this is okay to be left as public.

**Review by cijothomas — APPROVED** [review]

File: src/OpenTelemetry.Api/Internal/SpanHelper.cs (hunk 1 above)
Comment (utpilla) [reply src/OpenTelemetry.Api/Internal/SpanHelper.cs]: Updated this to public.

File: src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,8 @@
 * AspNetInstrumentation sets ActivitySource to activities created outside
   ActivitySource.
   ([#1515](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1515/))
+* Added SpanHelper class as a link to the the project.
```
Comment (CodeBlanch) [inline src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md:8]: Should this be in CHANGELOG?

Comment (cijothomas) [reply src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md:8]: we only need in APIs changelog, not in instrumentations. will do a cleanup

**Pull request merged**